app/Http/Transformers/Three/SignupTransformer.php

```diff
@@ -25,18 +25,23 @@ class SignupTransformer extends TransformerAbstract
      */
     public function transform(Signup $signup)
     {
-        return [
+        $response = [
             'id' => $signup->id,
             'northstar_id' => $signup->northstar_id,
             'campaign_id' => $signup->campaign_id,
             'campaign_run_id' => $signup->campaign_run_id,
             'quantity' => $signup->getQuantity(),
-            'why_participated' => $signup->why_participated,
-            'source' => $signup->source,
             'details' => $signup->details,
```

Comment:
I forget what this column is used for. Should it be hidden?

Reply:
@DFurnes I think for some potential affiliate stuff?

Reply:
yea right now it is only used to specify if the user opted into affiliate messaging. I think the argument for keeping it general was that eventually it could give more details about the signup source (like broadcast id) and such. I can see an argument for hiding it, since we are hiding source. But I don't think it is sensitive data. However

Reply:
Ohhhh right, for the opt-ins! It might make sense to hide as well when we have a chance.

Reply:
I can do this quickly now in a PR since it's top of mind!

Reply:
fix in this PR :) https://github.com/DoSomething/rogue/pull/610